di-ipv-orchestrator-stub/deploy/template.yaml

```diff
@@ -153,9 +153,9 @@ Mappings:
       platform: stubs
       dynatraceSecretArn: arn:aws:secretsmanager:eu-west-2:216552277552:secret:DynatraceNonProductionVariables
     "388905755587": #Stubs Prod
-      fargateCPUsize: "4096"
-      fargateRAMsize: "8192"
-      desiredTaskCount: 1
+      fargateCPUsize: "2048"
+      fargateRAMsize: "4096"
+      desiredTaskCount: 4
```

Comment:
Back of the envelope says this will change run cost from ~£100/month -> ~£200/month

We might want autoscaling or a way to manually scale for the duration of a load test, but I thought it better to try this first.